generators/wizard/templates/i18n/wizard.default.ts

```diff
@@ -4,7 +4,7 @@ export default {
     TITLE: "Wizard title",
     ACTION: i18nControls.actions.FINISH,
     steps: {
-        step1: {
+        step1: { // TODO Change this "step1" name to the wizard's step name
```

Comment:
There is no need for this TODO here, since someone who looks at this file will probably alter the Step 1 title. It may be better to change the "step 1 title" to "Details"

Reply:
This is a dangerous string, as this title will never show to the user, on a wizard that has only 1 step. It will only show if the wizard fails, and you have 2 steps ("Step 1" and "Results"). As such I believe it is safer to have a generic title instead of the current one

Reply:
I agree.